includes/fields/class-field-url.php

```diff
@@ -136,18 +137,12 @@ public function render_field_data( $data, $field ) {
 
         ob_start();
         ?>
-            <li class="<?php echo esc_attr( implode( ' ' , $container_classnames ) );  ?>">
-                <?php if ( ! $hide_label ): ?>
+            <li class="<?php echo esc_attr( implode( ' ', $container_classnames ) ); ?>">
+                <?php if ( ! $hide_label ) : ?>
                     <label><?php echo esc_html( $field['label'] ); ?>:</label>
                 <?php endif; ?>
-                <?php
-                    echo wp_kses( make_clickable( esc_url_raw( $data ) ), [
-                        'a' => [
-                            'href' => [],
-                            'rel'  => []
-                        ],
-                    ] );
-                ?>
+                <a href="<?php echo esc_url_raw( $data ); ?>"
+                   target="<?php echo $field['open_window'] === 'new' ? '_blank' : ''; ?>"> <?php echo esc_url_raw( $data ); ?> </a>
```

Comment:
Should you check if open_window is set or not empty? I'm not sure if it's a definitive value. Also there are empty spaces inside the tag. When it's a _blank target, WP adds a rel attribute with "noreferrer noopener" value. We should also add that for SEO purpose.

https://www.wpbeginner.com/beginners-guide/what-is-relnoopener-in-wordpress-explained/